test/acquisition/test_max_value_entropy_search.py

```diff
@@ -286,6 +287,51 @@ def test_q_multi_fidelity_max_value_entropy(self):
             X = torch.rand(1, 2, device=self.device, dtype=dtype)
             self.assertEqual(qMF_MVE(X).shape, torch.Size([1]))
 
+    def test_q_multi_fidelity_lower_bound_max_value_entropy(self):
```

Comment:
If this is exactly the same test as `test_q_multi_fidelity_max_value_entropy`, how about changing that function signature to 
```
def test_q_multi_fidelity_max_value_entropy(self, aqcf_class=qMultiFidelityMaxValueEntropy):
   ...
```
and using `aqcf_class` in the test. Then you can just do 
```
def test_q_multi_fidelity_lower_bound_max_value_entropy(self):
    self.test_q_multi_fidelity_max_value_entropy(acqf_class=qMultiFidelityLowerBoundMaxValueEntropy)
```
and avoid a bunch of code duplication.